.goreleaser.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+env:
+    # Global env vars for Go build.
+    - 'CGO_ENABLED=0'
+    - 'GO111MODULE=on'
+    - 'GOPROXY=https://proxy.golang.org,direct'
+  
+before:
+  hooks:
+    - 'go mod tidy'
+
+builds:
+  -
+    id: 'jiraplugin'
+    main: './cmd/jiraplugin'
+    binary: 'jiraplugin'
```

Comment:
The binary should match this pattern https://github.com/abcxyz/jvs/blob/main/pkg/plugin/plugin_manager.go#L31C16-L31C26

`jvs-plugin-jira`

Reply:
Done